src/main/java/org/sagebionetworks/bridge/services/ParticipantDataService.java

```diff
@@ -0,0 +1,75 @@
+package org.sagebionetworks.bridge.services;
+
+import org.sagebionetworks.bridge.BridgeConstants;
+import org.sagebionetworks.bridge.dao.ParticipantDataDao;
+import org.sagebionetworks.bridge.exceptions.BadRequestException;
+import org.sagebionetworks.bridge.models.ForwardCursorPagedResourceList;
+import org.sagebionetworks.bridge.models.ParticipantData;
+import org.sagebionetworks.bridge.models.reports.ReportDataKey;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.sagebionetworks.bridge.BridgeConstants.API_MAXIMUM_PAGE_SIZE;
+import static org.sagebionetworks.bridge.BridgeConstants.API_MINIMUM_PAGE_SIZE;
+
+@Component
+public class ParticipantDataService {
+    private ParticipantDataDao participantDataDao;
```

Comment:
Some things to check in this class:
1) is the user ID real and does the caller have access to it (you may do this in the controller however)
2) what if the identifier does not exist? (throw EntityNotFoundException)
3) what if pageSize is not provided?
4) what happens if someone saves data that has been saved already, leading to a version field mismatch? Do we throw a Bridge exception when this happens?